src/soorgeon/clean.py

```diff
@@ -0,0 +1,39 @@
+import subprocess
+import isort
+import click
+import tempfile
+import jupytext
+
+
+def basic_clean(task_file):
+    """
+    Run basic clean (directly called by cli.clean())
+    Generate intermediate files for ipynb
+    """
+    if task_file.lower().endswith(".ipynb"):
+        click.echo("Generating intermadiate py files for notebook cleaning")
+        nb = jupytext.read(task_file)
+        temp_path = tempfile.NamedTemporaryFile(suffix=".py",
+                                                delete=False).name
+        jupytext.write(nb, temp_path)
+        basic_clean_py(temp_path)
+        jupytext.write(jupytext.read(temp_path), task_file)
+    else:
+        basic_clean_py(task_file)
+    click.echo(f"Finished cleaning {task_file}")
+
+
+def basic_clean_py(task_file_py):
+    """
+    Run basic clean for py files
+    (util method only called by basic_clean())
+    """
+    # commands to execute, which follow <exectutable> <filename> format
+    commands = ["black"]
+    for command in commands:
+        result = subprocess.run([command, task_file_py],
+                                text=True,
+                                capture_output=True)
+        click.echo(result)
```

Comment:
please only print the confirmation message from black. Looks like it's being stored in `result.stderr`, otherwise, the output looks too verbose:

```
Generating intermadiate py files for notebook cleaning
CompletedProcess(args=['black', '/var/folders/3h/_lvh_w_x5g30rrjzb_xnn2j80000gq/T/tmp93a6zcba.py'], returncode=0, stdout='', stderr='All done! ✨ 🍰 ✨\n1 file left unchanged.\n')
```

Also, there's a chance the user doesn't have black installed, let's check that before running subprocess.run

```python
import shutil
from soorgeon.exceptions import BaseException

if shutil.which('black') is None:
    raise BaseException('black is missing, please install it with:\npip install black\nand try again')
```